docs/modules/ROOT/pages/concepts/meters.adoc

```diff
@@ -1,6 +1,11 @@
 [[meters]]
 = Meters
 
+A `Meter` is the interface for collecting a set of measurements (which we individually call metrics) about your application.
+
 Micrometer supports a set of `Meter` primitives, including `Timer`, `Counter`, `Gauge`, `DistributionSummary`, `LongTaskTimer`, `FunctionCounter`, `FunctionTimer`, and `TimeGauge`. Different meter types result in a different number of time series metrics. For example, while there is a single metric that represents a `Gauge`, a `Timer` measures both the count of timed events and the total time of all timed events.
 
+TIP: Recoding a measurement for a `Meter` is expected to be a relatively cheap operation and should not throw any exception.
+If the registry supports publishing metrics to a monitoring system, this is done in a separate thread snd should not affect recording metrics.
```

Comment:
It would probably be good to link to the registry page when mentioning "registry" here since we haven't introduced the concept yet now that this page comes before the registry one in the contents.

Reply:
Good point. I've added a link.